oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+autoAssign() {
+  echo "Current oncall is $1"
+  response=`curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $2" https://api.github.com/repos/microsoft/Oryx/issues | jq '.[]| select(.assignee == null or .assignee == "")' | jq '.number'`
```

Comment:
Is this getting all issues in our repository that don't have an assignee?

Since we know that this workflow is being triggered _only_ when an issue has been (re)opened, we should be able to assign the current on-call _only_ to the corresponding issue.

From the workflow file itself, we should be able to use [`${{ github.event.issue.number }}`](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#issues) to get the issue number, which we can pass to this script and use in this `autoAssign` function one time rather than iterating over each unassigned issue.

Reply:
I think this one is getting the first 30 open issues.
I'm not very familiar with the webhook, but i'll look into it a bit more. Thanks for the suggestion!